package.json

```
@@ -43,16 +43,26 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "@types/chalk": "^2.2.0",
+    "@types/common-tags": "^1.8.0",
+    "@types/listr": "^0.14.2",
+    "@typescript-eslint/eslint-plugin": "^2.5.0",
+    "@typescript-eslint/parser": "^2.5.0",
     "babel-plugin-htmlbars-inline-precompile": "^3.0.0",
+    "chalk": "^2.4.2",
+    "common-tags": "^1.8.0",
     "coveralls": "^3.0.6",
     "eslint": "^6.5.1",
     "eslint-config-prettier": "^6.3.0",
     "eslint-plugin-prettier": "^3.1.1",
     "execa": "^2.0.5",
     "jest": "^24.9.0",
+    "listr": "^0.14.3",
     "prettier": "^1.18.2",
     "release-it": "^12.4.2",
-    "release-it-lerna-changelog": "^1.0.3"
+    "release-it-lerna-changelog": "^1.0.3",
+    "ts-node": "^8.4.1",
+    "typescript": "^3.6.4"
```

Should generally never use `^` with `typescript`, it does not follow semver and generally speaking each minor version bump includes breaking changes:

```suggestion
    "typescript": "~3.6.4"
```